### ISSUE TITLE
Landing Page Wireframe

### ISSUE THREAD

**Issue by dk4jo3** [bigpanther/bigpanther.ca#1]
Hi all, 

I created some mockups for the BigPanther and ShipPanther landing pages. Please let me know what you think. It would be great if you all can help to brainstorm the copy content for the pages. [https://www.figma.com/file/Sq1Z5mCpNxHrezRTuirBDc/BigPanther-Wireframe?node-id=0%3A1](https://www.figma.com/file/Sq1Z5mCpNxHrezRTuirBDc/BigPanther-Wireframe?node-id=0%3A1)
If you click on the speech bubble icon on the top left, you can read the comments I left on the design. 

BigPather Landing Page mainly. 
1. Showcases different products and allow users to either download (ios/android) or learn more about the products (link to the products' landing pages) 
2. Showcases the mission statement of BigPanther and the contact information. 

ShipPanther Landing Page 
1. Showcases the features of the ShipPanther application
2. includes a FAQ section and contact us form. 

Please let me know if you have any feedback about the wireframe. I will begin to find illustrations for the page. 

I can apply the color scheme @martareg designer for the ShipPanther app to the website to be consistent. For the BigPanther page, do you guys have any colors in mind?

**Comment by harsimranmaan** [comment]
Thanks @dk4jo3. It would nice if we could use the similar colors are the app.

**Comment by martareg** [comment]
Hello, I will analyze your mockups and let you know maybe tomorrow :)

**Comment by martareg** [comment]
1. The first major question is, what is the purpose of a landing page, which is, what we want the landin page to convert?  If it is a Shipanther app download, then this CTA should be visible and easy for the user to click immediately after landing on the page.
The headline tells visitors exactly what the content is about for example:  Perfect shipment.

2. In my opinion, this feature section is a bit too long. Usually the user will read the first lines and only scan the other things. I would leave only the most important features here and if anyone is interested in the details, it's a good idea to add a 'Read More' button.
This is my suggestion.

![Web 1920 – 1](https://user-images.githubusercontent.com/77104115/106577221-657cfd80-653e-11eb-8f8f-c676db461514.png)

**Comment by dk4jo3** [comment]
Great feedback. 

Are you talking about the length of the text? or the number of the features in the feature section? 3-4 features are a perfect amount for an app like ShipPanther and I agree that we can workshop the copy content to keep it specific and concise. 

I would disagree with the need for the "scroll down to find our more" hint. Users naturally scroll down ([reference](https://uxmyths.com/post/654047943/myth-people-dont-scroll)) and the next section would be slightly above the fold.

And about the "read more" link, I don't think they would be necessary considering the features and benefits of the apps are not too complicated. But we can add a 'how it works' page to demonstrate the process and I agree that the "read more..." would be applicable if the app expands its features in the future.

The main purpose of the respective landing pages would be:
1. BigPanther: introduce the products under bigPanther and the team/mission statement, encourage the users to check out the products' pages.
2. ShipPanther: encourage the users to download the app by showcasing the features and benefits of the application. (secondary purpose would be answering common questions and show them how the app would work)

**Comment by martareg** [comment]
Yes I mean about the length of the text  so that the content is concise and specific, btw good job :)

**Comment by dk4jo3** [comment]
Made update of the ShipPanther Page mockup. Now it has some illustrations from freepik.com. Attributions for them are at the footer.
Please take a look and let me know about the color scheme. 
[https://www.figma.com/file/Sq1Z5mCpNxHrezRTuirBDc/BigPanther-Wireframe?node-id=0%3A1](https://www.figma.com/file/Sq1Z5mCpNxHrezRTuirBDc/BigPanther-Wireframe?node-id=0%3A1
)

What do you guys think the color scheme for the Big Panther page should be?
